packages/adapters-api/src/db-service.ts

```diff
@@ -57,33 +71,64 @@ export class PostgresService implements DbService {
     this.#dbPools = dbPools
   }
 
-  async getAddressPools(
+  async getDefiPositionsDetectionPerChain(
     userAddress: string,
   ): Promise<Partial<Record<EvmChain, string[]>>> {
     return Object.values(EvmChain).reduce(
```

Comment:
Nothing to do with this PR, but I've just realised that this construct will run the loop of `getAddressChainPools` sequentially, not in parallel.

It's probably not worth worrying about it, but if that is the case maybe we should get the results for each chain as a map with a `Promise.all` and then build the object.